model_analyzer/config/input/config_command.py

```diff
@@ -113,6 +113,8 @@ def _load_yaml_config(self, args: Namespace) -> Optional[Dict[str, List]]:
     def _check_for_illegal_config_settings(
             self, args: Namespace, yaml_config: Optional[Dict[str,
                                                               List]]) -> None:
+        # Illegal config settings check for ensemble is done in model_manager,
```

Comment:
This implies it is a comment for the lines below it.
Can you just start it with "Note: Illegal ..." so it is clear it is a standalone note comment, and then add an empty line before the other code?

Reply:
Done